deploy/olm-catalog/node-maintenance-operator/manifests/node-maintenance-operator.clusterserviceversion.yaml

```diff
@@ -31,10 +31,16 @@ spec:
       kind: NodeMaintenance
       name: nodemaintenances.nodemaintenance.kubevirt.io
       version: v1beta1
-  description: Node maintenance operator
+  description: |
+    Node Maintenance Operator
+
+    This operator will keep nodes cordoned and drained while a matching node maintenance CR exists.
+    This is useful when investigating problems with a machine, or performing an operation on the underlying failure that might result in node failure.
```

Comment:
s/underlying failure/underlying machine?

Reply:
hm, not sure... :thinking: 
@beekhof since I copied from your Comet description: what is correct here?

Reply:
machine.

probably a brain fart on my part